docs/installation-guide-edge.md

```diff
@@ -0,0 +1,120 @@
+  > 📝 Kubernetes < `v1.21.x` is needed due to [`no matches for kind "CustomResourceDefinition error`](https://github.com/kubeflow/manifests/issues/2028)
+* [kubectl installed](https://kubernetes.io/docs/tasks/tools/#kubectl) and [configured](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) to use the kubernetes cluster.
+* [kustomize installed](https://kubectl.docs.kubernetes.io/installation/kustomize/).
+* [curl](https://curl.se/download.html).
+* [python3](https://www.python.org/downloads/): To fetch the latest release tag.
+* The [fl-suite repository](https://github.com/katulu-io/fl-suite) cloned and current working directory in `/path/to/cloned/fl-suite/docs`
+
+## Deploy a fl-edge
+
+### 1. Download the fl-edge manifests:
+
+```shell
+LATEST_RELEASE_URL=$(curl -s "https://api.github.com/repos/katulu-io/fl-suite/releases/latest")
+FL_EDGE_MANIFESTS_URL=$(echo $LATEST_RELEASE_URL | python3 -c 'import json,sys;print(json.load(sys.stdin)["assets"][0]["url"])')
+mkdir -p release-manifests/fl-edge && curl -s -L -H "Accept: application/octet-stream" "$FL_EDGE_MANIFESTS_URL" | tar xzf - --strip-components=1 -C release-manifests/fl-edge
+```
+
+### 2. Install the fl-edge manifests
+
+```shell
+while ! kustomize build example/fl-edge | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
+```
+
+Wait for all the pods to be ready:
+
+```shell
+kubectl get pods -n fl-operator-system
+```
+
+### 3. Create a new edge-identity
+
+Go to the running instance of the fl-suite and create a new edge:
+
+![](../images/fl-suite-menu.png)
+<br />
+
+![](../images/fl-suite-edges.png)
+<br />
+
+![](../images/fl-suite-edge-form.png)
+<br />
+
+### 4. Deploy the edge-identity CRD
+
+Copy the CRDs from the new edge by pressing the *Copy* button:
+
+![](../images/fl-suite-new-edge.png)
+<br />
+
+Apply the CRD into the fl-edge kubernetes cluster:
+
+```shell
+kubectl apply -f - <<EOF
+---
+apiVersion: fl.katulu.io/v1alpha1
+kind: FlEdge
+metadata:
+  name: my-edge
+spec:
+  auth:
+    spire:
+      server-address: spire-ger-lsxsrwv0.demo.fl.katulu.io
+      server-port: 443
+      trust-domain: katulu.io
+      join-token: fdd706ed-0a28-a999-21b08bba618f
+      skip-kubelet-verification: true
+---
+apiVersion: fl.katulu.io/v1alpha1
+kind: FlOperator
+metadata:
+  name: my-edge
+spec:
+  orchestrator-url: spire-ger-lsxsrwv0.demo.fl.katulu.io
+  orchestrator-port: 443
+  orchestrator-sni: fl-orchestrator-ger-lsxsrwv0.demo.fl.katulu.io
+  registry-credentials:
+    secret: regcred
+---
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/dockerconfigjson
+metadata:
+  name: regcred
+data:
+  .dockerconfigjson: eyJhdXRocyI6eyJodHRwczovL3JlZ2lzdHJ5LWdlci1sc3hzcnd2MC5kZW1vLmZsLmthdHVsdS5pbyI6eyJhdXRoIjoiYTJGMGRXeDFMV1pzT2t0M1pXVTNOMEpRYUcxcU5GbENPV2xMYVVzNFpsRnlRVWswYVdGb2JXMVYifX19
```

Comment:
This is a bit misleading, as this could lead users to copy-paste this exact command. Let's be a bit more general and explain to apply the resources copied before, e.g.

```
kubectl apply -f - <<EOF
(paste resources here)
EOF

Reply:
Good idea. I'll change it.